vip-config/vip-config.php

```diff
@@ -60,3 +60,14 @@
 if ( function_exists( 'newrelic_disable_autorum' ) ) {
 	newrelic_disable_autorum();
 }
+
+/**
+ * Set WP_DEBUG to true for all non-production environments to ensure _doing_it_wrong() notices display
+ * in Query Monitor. This also changes the error_reporting level to E_ALL.
+ *
+ * @see https://wordpress.org/support/article/debugging-in-wordpress/#wp_debug
+ */
+
+if ( defined( 'VIP_GO_APP_ENVIRONMENT' ) && 'production' !== VIP_GO_APP_ENVIRONMENT ) {
```

Comment:
Could we amend this condition to also set WP_DEBUG when the constant isn't defined, so that it enables it automatically for local development as well?

Reply:
```suggestion
if (
	( defined( 'VIP_GO_APP_ENVIRONMENT' ) && 'production' !== VIP_GO_APP_ENVIRONMENT ) ||
	! defined( 'VIP_GO_APP_ENVIRONMENT' )
) {
```

Needs testing, and the DocBlock updated accordingly.